content/tutorial/tutorial.md

```diff
@@ -213,22 +218,23 @@ class Square extends React.Component {
 }
 ```
 
-Before:
+पहले:
 
 ![React Devtools](../images/tutorial/tictac-empty.png)
 
-After: You should see a number in each square in the rendered output.
+बादमे: आपको प्रदान किए गए आउटपुट में प्रत्येक वर्ग में एक संख्या दिखनी चाहिए।
 
 ![React Devtools](../images/tutorial/tictac-numbers.png)
 
-**[View the full code at this point](https://codepen.io/gaearon/pen/aWWQOG?editors=0010)**
+**[इस बिंदु पर पूर्ण कोड देखें](https://codepen.io/gaearon/pen/aWWQOG?editors=0010)**
+
 
-Congratulations! You've just "passed a prop" from a parent Board component to a child Square component. Passing props is how information flows in React apps, from parents to children.
+बधाई हो! आपने अभि "एक props" पैरेंट Board कौम्पोनॅन्ट से एक चाइल्ड Square कौम्पोनॅन्ट को पास किया है। Props पासिंग यह है कि कैसे React एप्स में पैरेंट से लेकर चाइल्ड तक की जानकारी प्रवाहित होती है।
```

Comment:
अभि -> badi maatra

Reply:
Props पासिंग यह है कि कैसे React एप्स में पैरेंट से लेकर चाइल्ड तक की जानकारी प्रवाहित होती है। -> grammar check?

Reply:
`Props पासिंग यह है कि, कैसे React एप्स में पैरेंट से लेकर चाइल्ड तक जानकारी kaise jati hai` ?

Reply:
Props पासिंग se humara matlab hai ki would be better?

Reply:
humara matlab yeh hai ki